src/status_im/signals/core.cljs

```diff
@@ -50,6 +51,20 @@
     {:db (assoc db :peer-stats peer-stats
                 :peers-count (count (:peers peer-stats)))}))
 
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide])
+  (re-frame/dispatch event))
+
+(defn handle-local-pairing-signals [signal-type]
+  (log/info "localpairing signal received ====> " signal-type)
+  ;  (if (= signal-type "process-success") ;; samyoul to make changes to signal name
+  ;    (log/info "inside process success workflow")
+  ;    ;; TODO : when status-go has this capability then,
+  ;    ;; we should ideally navigate to multi accounts after identifying a signal that says that process is sucessful
```

Comment:
consider removing or simplifying these comments. A lot of these seems like it should not be in the codebase.

Reply:
kept this code because I may or may not want to use it in the future but will definitely clean this up in the follow up PRs

Reply:
Please @siddarthkay, if you can, remove all dead code from this PR, leave these things only in your local environment (branches, stashes, etc). It's the same discussion we had in a thread at the top https://github.com/status-im/status-mobile/pull/14514/files#r1044552281

Reply:
I mean, you could also push a remote branch so it's there for reference too.